docs/types/traits-and-interfaces.md

```diff
@@ -62,17 +98,23 @@ Here, we have a class `Larry` that has a `name` function with the same signature
 
 __Wait, why not?__ Because `Larry` doesn't say it `is Named`. Remember, traits are __nominal__: a type that wants to provide a trait has to explicitly declare that it does. And `Larry` doesn't.
 
-## Structural subtyping
+You can also do nominal subtyping using the keyword `interface`. __Interfaces__ in Pony are primarily used for structural subtyping. However, interfaces, like traits can have default method implementations and in order to use default method implementations, an interface must be used in a nominal fashion. For example:
```

Comment:
I would move this paragraph and its example to the "Structural subtyping" section, since both focus on `interfaces`. I've also suggested some changes to the last part of the paragraph, since I found the word a bit confusing.

```suggestion
You can also do nominal subtyping using the keyword `interface`. __Interfaces__ in Pony are primarily used for structural subtyping. Like traits, interfaces can also have default method implementations, but in order to use default method implementations, an interface must be used in a nominal fashion. For example:
```

Reply:
I disagree with the moving. I think it is best to introduce both concepts then cover them particularly as the interface keyword allows for a bit of nominal as well as structural.